src/components/common/ArbitratorDropdown/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+import { ButtonSelect } from '../../buttons/ButtonSelect'
+import { SelectItem } from '../../form/SelectItem'
+import { Dropdown, DropdownPosition } from '../Dropdown'
+
+interface Props {
+  onClick: (value: string) => void
+  value: string
+}
+
+export const ArbitratorDropdown = ({ onClick, value }: Props) => {
+  const arbitratorItems = [
+    {
+      text: 'Realit.io',
```

Comment:
This could be improved, if this PR will be merge https://github.com/gnosis/conditional-tokens-explorer/pull/105

Reply:
We can  improve typings, and some other stuff